### PR TITLE
Add `alpha` option

### LINKED ISSUE #17: Support optional alpha value
related at #14 

I want optional alpha value on `hexRgb.Options`.
Can i contribute this project?

### PR DESCRIPTION
**Pull request by juyeong1260**
resolved #17

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -8,6 +8,8 @@ declare namespace hexRgb {
 		@default 'object'
 		*/
 		readonly format?: 'object' | 'array' | 'css';
+
+		readonly alpha?: number;
```
Comment (sindresorhus) [inline index.d.ts:19]: Needs a doc comment. Should be in sync with the readme.

File: readme.md [hunk 2]
```diff
@@ -65,6 +71,13 @@ The RGB output format.
 
 Note that when using the `css` format, the value of the alpha channel is rounded to two decimal places.
 
+##### alpha
+
+Type: `number`
+
+The Alpha value to be applied with the first priority.
```
Comment (sindresorhus) [inline readme.md]: This could be better written.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

File: readme.md (hunk 2 above)
Comment (juyeong1260) [reply readme.md]: I'm sorry for not speaking English well..

I applied your review on cce9864. Please make sure it's okay again

File: readme.md [hunk 3]
```diff
@@ -75,7 +75,8 @@ Note that when using the `css` format, the value of the alpha channel is rounded
 
 Type: `number`
 
-The Alpha value to be applied with the first priority.
+Note that when 'alpha' option is added, the value of alpha in the hex code is overwritten.
```
Comment (sindresorhus) [inline readme.md]:
```suggestion
Set the alpha of the color.

This overrides any existing alpha component in the Hex color string. For example, the `99` in `#22222299`.
```

**Pull request merged**